src/instructlab/model/train.py

```diff
@@ -641,7 +642,14 @@ def get_files(directory: str, pattern: str) -> list[str]:
 
             train_args, torch_args = map_train_to_library(ctx, ctx.params)
 
-            run_training(train_args=train_args, torch_args=torch_args)
+            try:
+                run_training(train_args=train_args, torch_args=torch_args)
+            except RuntimeError as e:
+                # `run_training` currently throws `RuntimeError` to indicate a
+                # failure that occurred inside of the training loop
+                logger.error("Error occurred inside of the training loop.", exc_info=e)
+                sys.exit(1)
```

Comment:
I think `raise click.exceptions.Exit(1)` is preferred

Reply:
Was not aware of this API, thank you!